utils.groovy

```diff
@@ -301,7 +301,7 @@ def build_artifacts(pipecfg, stream, basearch) {
 
     // Define the parallel jobs in a map
     parallelruns = artifacts.collectEntries {
-        [it, { shwrap("cosa buildextend-${it}") }]
+        ["💽:${it}", { shwrap("cosa buildextend-${it}") }]
```

Comment:
Minor: not sure if it's the Firefox rendering, but here there's no space before the colon while in the cloud upload there is.

Reply:
yeah I think it's the choice of emoji. The intention is for the emojis to not have space between them. Let me do a fixup for the other ones in libupload.groovy.

Reply:
well then it makes it look weird in VIM. I'm just going to leave it like it is. Let me know how it looks on the other end if we want to change it.